src/classes/RD2_RecurringDonation.cls

```diff
@@ -352,4 +363,11 @@ public inherited sharing class RD2_RecurringDonation {
         }
         return false;
     }
+
+    /***
+    * @description Contains attribute confirming if Elevate Integration is not enabled
+    */
+    public Boolean isElevateIntegrationDisabled() {
```

Comment:
The `RD2_RecurringDonation` is the wrapper on the RD SObject, so the answers it can provide are related to the RD record mostly, ie its properties are not related to any integration/configuration. So the following method can be 
```

    /***
    * @description Checks if the Recurring Donation is originated in Elevate
    */
    public Boolean isElevateRecord() {
       return String.isNotBlank(rd.CommitmentId__c);
    }
```

Since this property confirms the record Elevate origin, see what needs to be changed in `RD2_OpportunityEvaluationService` which currently checks if the Elevate integration is disabled.